tests/main/snaps-state/task.yaml

```diff
@@ -74,3 +74,56 @@ execute: |
         snap remove "$SNAP_JAILMODE"
         rm -f "$TESTSLIB/snaps/${SNAP_JAILMODE}/${SNAP_JAILMODE}_1.0_all.snap"
     fi
+
+    # check the command is-confinement-supported works
+    case "$SPREAD_SYSTEM" in
+        fedora-*)
+            not "$TESTSTOOLS"/snaps-state is-confinement-supported classic
+            not "$TESTSTOOLS"/snaps-state is-confinement-supported strict
+            "$TESTSTOOLS"/snaps-state is-confinement-supported devmode
+            ;;
+        ubuntu-core-*)
+            not "$TESTSTOOLS"/snaps-state is-confinement-supported classic
+            "$TESTSTOOLS"/snaps-state is-confinement-supported strict
+            "$TESTSTOOLS"/snaps-state is-confinement-supported devmode
+            ;;
+        ubuntu-1*|ubuntu-2*)
+            "$TESTSTOOLS"/snaps-state is-confinement-supported classic
+            "$TESTSTOOLS"/snaps-state is-confinement-supported strict
+            "$TESTSTOOLS"/snaps-state is-confinement-supported devmode
+            ;;
+    esac
+
+    # Check the repack of the snapd deb into core and snapd snaps
+    "$TESTSTOOLS"/snaps-state repack-snapd-deb-into-snap core
+    test -e core/core-from-snapd-deb.snap
+    rm -rf core
+
+    "$TESTSTOOLS"/snaps-state repack-snapd-deb-into-snap snapd
+    test -e snapd/snapd-from-deb.snap
+    rm -rf snapd
+
+    cp "$SPREAD_PATH"/../snapd_*.deb current_snapd.deb
```

Comment:
Where do we get snapd_*.deb from on Fedora?

Reply:
fixed